src/material/tooltip/testing/tooltip-harness.ts

```diff
@@ -52,6 +52,11 @@ export abstract class _MatTooltipHarnessBase extends ComponentHarness {
     return !!panel && !(await panel.hasClass(this._hiddenClass));
   }
 
+  /** Gets whether the tooltip is disabled */
+  async isDisabled(): Promise<boolean> {
+    return !(await this._optionalPanel());
```

Comment:
Would this be `true` if the user did not first call `show()`? Perhaps this method should attempt to show the tooltip, see if it's there, then hide it afterwards in case it wasn't disabled

Reply:
Yeah, you're right. 👍 

Btw, the reason I didn't call `show()` inside here is because I saw a method like `getTooltipText()` requires the user to call `show()` otherwise it will return an empty string.(maybe `getTooltipText()` should also handle show/hide in itself to provide better DX)